ui/raidboss/data/02-arr/raid/t6.js

```diff
@@ -169,10 +166,7 @@
       regexJa: Regexes.startsUsing({ id: '86C', source: 'ラフレシア', capture: false }),
       regexCn: Regexes.startsUsing({ id: '86C', source: '大王花', capture: false }),
       regexKo: Regexes.startsUsing({ id: '86C', source: '라플레시아', capture: false }),
-      infoText: {
-        en: 'Stack for Acid',
-        fr: 'Packez vous pour Acide',
-      },
+      response: Responses.stack(),
```

Comment:
I would leave this.  Maybe I'm overthinking, but I think it's nice to differentiate stack (shared damage marker) with bait (stack up to drop ground aoes)? "Stack for acid" at least indicates why?

Reply:
ok i will revert it